### DISCUSSION TITLE
Feature request: Tilføj skoleskema som calendar entities

### DISCUSSION THREAD

**Discussion by scaarup** [JBoye/HA-Aula#18] [General]
Tilføj skoleskema som calendar entities. Der er en kalender i Aula:

'https://www.aula.dk/api/v14/?method=calendar.getEventsByProfileIdsAndResourceIds'

Der skal spørges på på childids og en tidsperiode. Eksempel post data:
'{"instProfileIds":[1356736,1378381],"resourceIds":[],"start":"2022-10-10 00:00:00.0000+02:00","end":"2022-10-15 00:00:00.0000+02:00"}'

**Comment by JBoye** [comment]
God ide, jeg kigger på det om 4 års tid når min datter starter i folkeskolen ;-)

**Comment by scaarup** [comment]
> God ide, jeg kigger på det om 4 års tid når min datter starter i folkeskolen ;-)

Jeg er selv ved at kigge lidt på det, så der kommer nok et pr på et tidspunkt 🙂

**Comment by Jogge** [comment]
@scaarup Jeg ser du har lavet et nyt projekt her: https://github.com/scaarup/aula

Er det fordi dette repo har været for inaktivt og sløv til at acceptere PR? 🙂

**Comment by scaarup** [comment]
> @scaarup Jeg ser du har lavet et nyt projekt her: https://github.com/scaarup/aula
> 
> Er det fordi dette repo har været for inaktivt og sløv til at acceptere PR? 🙂

Ja lidt :). Jeg havde "tusind" idéer og selv de mindste af dem, kneb det med at få igennem her. Så tænkte det var meget nemmere og hurtigere at starte lidt på en frisk ;)

**Comment by Jogge** [comment]
@scaarup Jeg giver dit projekt et skud snarest :)

**Comment by scaarup** [comment]
Det må du meget gerne 🙂. Jeg er den eneste, der har testet det indtil videre og kunne godt bruge lidt feedback.

**Comment by JBoye** [comment]
(Jeg har lige lavet en discussion i stedet for issue)

Jeg har været lidt tilbageholdende for at bruge mere tid på dette projekt både fordi jeg har et ambivalent forhold til python, og fordi jeg har været i tvivl om projektets fremtid hvis Unilogin skulle blive helt lukket

Samtidig har jeg ikke skolebørn endnu, så mange af de features scaarup foreslår er ikke så relevant til mig. 

Men jeg synes at det er super fedt at Søren har lyst til at bygge videre på det, og det ser skidegodt ud indtil videre (har ikke selv testet endnu)

@scaarup Hvad siger du til at blive medejer af dette repo i stedet for? 
Det er jo dette repo som folk har installeret, så de ville kunne opgradere direkte

**Comment by scaarup** [comment]
Det er fuldt forståeligt.
Jeg er nu heller ikke super skarp i Python, men jeg kan godt lide det. Min baggrund er i IT operations og infrastruktur 🙂.
Nej, der vil ikke kunne opgraderes direkte. Jeg har lavet mit repo til et HACS repo og der er allerede en PR på at blive optaget i HACS. Desuden har jeg gjort den del ud af at sætte github actions osv op, så der er styr på releases, kode kvalitet osv. osv.
Men du må meget gerne komme over og blive contributer/owner (hvad det nu hedder).

**Reply by JBoye** [reply]
Det er helt fint 👍

Jeg begynder at anbefale at folk installerer din i stedet
